lib/nexpose/common.rb

```diff
@@ -154,11 +149,12 @@ class Schedule < APIObject
     attr_accessor :day
     attr_accessor :occurrence
     attr_accessor :start_month
+    # Timezone cannot be set, always console timezone. If console timezone is not supported defaults to utc.
     attr_accessor :timezone
```

Comment:
I'll reword this, timezone can be read back from the site schedule. But when creating a schedule adding a timezone will do nothing.